mpas_analysis/default.cfg

```diff
@@ -1564,22 +1564,25 @@ comparisonGrids = ['latlon', 'subpolar_north_atlantic']
 ## (if available)
 
 # colormap for model/observations
-colormapNameResult = blue-orange-div
+colormapNameResult = 3wave-blue-red-brown
 # whether the colormap is indexed or continuous
 colormapTypeResult = continuous
 # color indices into colormapName for filled contours
 # the type of norm used in the colormap
-normTypeResult = symLog
+normTypeResult = linear
 # A dictionary with keywords for the norm
```

Comment:
Could we maybe add a suggestion for Arctic plots, just as a comment? Something like: "Suggested values for Arctic focused analysis: vmin=-50; vmax=10; contourLevelsResult = np.arange(-50., 11.0, 5.)"

Reply:
We can add a section to `polar_regions.cfg` with your suggestion.  I think that's better than just having a suggestion here.

Reply:
@milenaveneziani, my concern about your suggested vmax and vim is that this doesn't work too well for a symmetric colormap like I was using.  Did you try that out and see that it works okay?

Reply:
Good idea Xylar. May I make some suggestions below then?

Reply:
I tried it, but with the `cmocean.cm.curl` colormap. I was going to suggest to use that for NH plots, for the reasons that you mentioned

Reply:
Sure!

Reply:
Maybe leave this alone for this PR, and I can try it on the develop branch that we are testing the Risk Index analysis on. I can get back to you after that.

Reply:
> I tried it, but with the cmocean.cm.curl colormap. I was going to suggest to use that for NH plots, for the reasons that you mentioned

I would really prefer not to use `curl` only because then it's clearer which are the corrected BSF plots and which are the old, incorrect ones.

Reply:
> Maybe leave this alone for this PR, and I can try it on the develop branch that we are testing the Risk Index analysis on. I can get back to you after that.

Well, I would kind of like to nail this down right away because we will get plots with whatever we decide on here for the next 6 to 8 months from E3SM-Unified.